### PR TITLE
Cochrane Fixes

### PR DESCRIPTION
**Pull request by FBartos**
fixes:
- 1 study error: set an informative error message when the selected data set contains only one estimate
- incompatible input for added estimates: the GUI now allows only SE or CI to be specified (ES & SE or frequencies in case of dichotomous data)

features:
- ability to export the selected data sets (https://twitter.com/Kuba_Ruszkowski/status/1531688333532966912)

### CODE REVIEW

**Comment by FBartos** [comment]
The windows unit failed because there was an update to the Bayesian Meta-Analysis code that changed the scaling of the x-axis in the figures.

There is one more error in the qml interface -- it shows an empty checkbox in the study selection but @boutinb is gonna fix it in another PR, so this is ready for review.

File: R/cochranecommon.R [hunk 1]
```diff
@@ -594,3 +608,44 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
...
+      tempDataset   <- dataset[dataset[,"titleMetaAnalysis"] %in% c("_add", title),]
+      write.csv(
+        x         = tempDataset,
+        file      = gsub(".csv", paste0("-", i, ".csv"), options[["savePath"]], fixed = TRUE),
+        row.names = FALSE
+      )
+    }
+
+  }else{
+    write.csv(
```
Comment (vandenman) [inline R/cochranecommon.R:642]:
```suggestion
    utils::write.csv(
```

File: R/cochranecommon.R [hunk 2]
```diff
@@ -594,3 +608,44 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
...
+  }
+
+  if (options[["analyzeData"]] == "individually") {
+    selection  <- unique(dataset[,"titleMetaAnalysis"])
+    selection  <- selection[selection != "_add"]
+
+    for (i in seq_along(selection)) {
+      title         <- sort(selection, decreasing = TRUE)[i]
+      tempDataset   <- dataset[dataset[,"titleMetaAnalysis"] %in% c("_add", title),]
+      write.csv(
```
Comment (vandenman) [inline R/cochranecommon.R:634]:
```suggestion
      utils::write.csv(
```

**Review by vandenman — APPROVED** [review]

File: R/cochranecommon.R (hunk 1 above)
Comment (vandenman) [inline R/cochranecommon.R:642]:
```suggestion
    utils::write.csv(
```

**Comment by vandenman** [comment]
feel free to merge once tests pass

**Pull request merged**